src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs

```diff
@@ -97,36 +96,20 @@ private void UpdatePageLayout()
 				foreach (var item in MainPropertiesViewModel.NavigationViewItems) item.IsCompact = false;
 		}
 
-		private async void AppSettings_ThemeModeChanged(object? sender, EventArgs e)
+		private async void AppThemeModeService_AppThemeModeChanged(object? sender, EventArgs e)
 		{
 			if (Parent is null)
 				return;
 
 			await DispatcherQueue.EnqueueOrInvokeAsync(() =>
 			{
-				((Frame)Parent).RequestedTheme = ThemeHelper.RootTheme;
-
-				switch (ThemeHelper.RootTheme)
```

Comment:
Are the title bar buttons still displayed correctly even if the theme is changed at runtime?

Reply:
Yes, absolutely.
When the Combo box invokes theme changed event, properties window also apply the theme through the same method without invoking theme changed event.